mthree/mitigation.py

```diff
@@ -307,7 +306,7 @@ def _grab_additional_cals(self, qubits, shots=None, method='balanced', rep_delay
             trans_qcs = transpile(circs, self.system, optimization_level=0)
 
         # This Backend check is here for Qiskit direct access.  Should be removed later.
-        if isinstance(self.system, Backend):
+        if self.system.__class__.__name__ == 'DirectBackend':
```

Comment:
This would work assuming the backends aren't subclasses. The other fix you could do is just

```suggestion
        If not isinstance(self.system, Backend):
```

As `BaseBackend` does not inherit from `Backend`

Reply:
Yeah that is likely better.  I was cautious of doing so because if Direct moves to `Backend` then that check fails.  Not sure if `Backend` works with `execute` and Qobj pathways though.  Perhaps can just worry about that when/if the time comes.